### PR TITLE
updates from beta to production

### PR DESCRIPTION
**Pull request by laurakarhu**
Updating documentation and component statuses to 1.0

## Description

Updated pre-release description to encourage feedback. Updated footer, spinner, tabs, image with card and section to stable. Removed mention of beta on the component overview page.

## Motivation and Context

Updating these is part of the 1.0 preparations, making sure documentation is clean and up to date.

## How Has This Been Tested?
Running docsite locally

### CODE REVIEW

File: site/docs/components/component_overview.mdx [hunk 1]
```diff
@@ -63,7 +63,7 @@ Component                                                        | Status
 Dropdown                                                         | <StatusLabel type="error">Deprecated</StatusLabel>   |                                                                               |
 [Footer](/components/footer)                                     | <StatusLabel type="alert">Pre-release</StatusLabel>  |                                                                               | [<IconCheck/>](/storybook/react/?path=/story/components-footer--default)
```
Comment (minevala) [inline site/docs/components/component_overview.mdx:64]: Footer is marked as stable in the component docs?

Comment (laurakarhu) [reply site/docs/components/component_overview.mdx:64]: Good catch. This was the one that we kept pondering if it's stable or not so of course it got half way there :D I'll fix it.

Comment (minevala) [reply site/docs/components/component_overview.mdx]: Stable or not? 🤔 
Is it time to toss a coin?

**Review by minevala — APPROVED** [review]
Nice 👍

**Comment by minevala** [comment]
Probably this PR needs other changes too? I was too fast with the approve button 😁

**Comment by laurakarhu** [comment]
> Probably this PR needs other changes too? I was too fast with the approve button 😁

You were on time, Github just took more than 30 mins to make the change :D I realized that at least on the component overview is one of the places we tell about beta so I thought I'd fix that in the same pr

**Comment by ronijaakkola** [comment]
Tab status labels are slightly indented for some reason.

image

**Comment by ronijaakkola** [comment]
The "New" label should probably be removed from the Dialog.

image

**Comment by ronijaakkola** [comment]
The "New" label should probably be removed from the DateInput.

image

**Comment by ronijaakkola** [comment]
Same indentation problem with loading spinner labels.

image

**Comment by ronijaakkola** [comment]
The "New" label should probably be removed from the TimeInput.

image

**Comment by ronijaakkola** [comment]
Made some changes:
- Removed "New" label from all components
- Changed "Pre-release" label to "Draft" label
- Changed form pattern statuses from "Pre-release" to "Stable"

**Comment by laurakarhu** [comment]
Today we discussed that footer has some issues so it's not stable yet. Also removed the Dropdown from the list.

**Review by ronijaakkola — APPROVED** [review]

**Pull request merged**